Android/ChorusRFLaptimer/app/src/main/java/app/andrey_voroshkov/chorus_laptimer/RaceSetupFragment.java

```diff
@@ -104,6 +107,7 @@ public void onDataChange(DataAction dataItemName) {
         Button btnIncAdjust = (Button) rootView.findViewById(R.id.btnIncAdjustmentConst);
         TextView txtVoltage = (TextView) rootView.findViewById(R.id.txtVoltage);
         LinearLayout layoutVoltage = (LinearLayout) rootView.findViewById(R.id.layoutVoltage);
+        CheckBox chkAlertNewBestLap = (CheckBox) rootView.findViewById(R.id.chkAlertNewBestTime);

```

Comment:
I think it should be dependent on "Speak Lap TImes" checkbox.  Because if "Speak Lap TImes" feature is disabled, you won't hear the Best lap as well. 
So you need to disable this checkbox whenever Speak lap Times is unchecked and enable when the latter is checked.
Also need to make sure it loads properly from preferences it all combinations.